### DISCUSSION TITLE
Latest windows 64bit update fails virus total malware scan twice.

### DISCUSSION THREAD

**Discussion by Bluto39** [aclap-dev/video-downloadhelper#1160] [Help and Q&A]
### Your question

Virus Total malware scanner flags your latest update from two separate scanning services. 
Any way to fix this? It exceeds my threat tolerance, as well it should, Open source or not.
Prior updates always passed this scan as safe so here's hoping this is not a wontfix.

![Scrnsht- 8_18_2024 , 12_23_56 PM](https://github.com/user-attachments/assets/59ec2bb6-6b64-4802-a362-aff540d0b6f1)

### Addon & CoApp version and details

n/a

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

There's a modest number of reports similar to yours in this forum.  You should hunt them down & read them.  If you downloaded the the VDH browser extension & the CoApp from the approved site, then you got the genuine, virus-free product.  The approved site is the Wiki of this forum, a link to which appears at the top of every page of this forum.  You are looking at false positives.  False positives are a routine occurrence in the antivirus world.  After all, antivirus software is exactly that, software.  Therefore it is prone to bugs & other shortcomings, same as any other software.  Antivirus software is no more sacrosanct than any other software.  Not every antivirus program can know about every piece of software released every day.  That's why every antivirus program includes a mechanism for telling the antivirus program that it has raised a false positive.  Just tell your antivirus program to let VDH & its CoApp pass.  Do not quarantine any part of either portion of the product.  In addition, you need to let the CoApp through any firewall you might be using.  It has been the experience of not just myself but many other users that the Windows Defender built into Windows by Microsoft has never raised the smallest peep about VDH & the CoApp.  That is the recommended antivirus/firewall that you should be using.  There is information about antivirus in the Wiki.  Be sure you find that & read it as well.
